### ISSUE TITLE
Multiple entry conditions

### ISSUE THREAD

**Issue by fvwanja** [spicywebau/craft-neo#810]
### Bug Description

![Screenshot 2023-10-26 at 12 48 05](https://github.com/spicywebau/craft-neo/assets/90781427/9d8d82dd-8b1b-4948-b149-7640eeefa9f5)

When I want to create multiple conditions for a field to show up like in the screenshot above, the second input field does not show up after saving. In the browsers console, I can see an error message:

![Screenshot 2023-10-26 at 12 51 21](https://github.com/spicywebau/craft-neo/assets/90781427/2c45c9c3-d3d1-4981-85b3-7ff018bf5a2d)

Uncaught ReferenceError: Vue is not defined
    at <anonymous>:138:1
    at m (jquery.js?v=1698215339:2:867)
    at Pe (jquery.js?v=1698215339:2:48869)
    at w.fn.init.append (jquery.js?v=1698215339:2:50073)
    at s.constructor.initUi (BlockTypeSettings.js:129:18)
    at s.constructor.addItem (Configurator.js:197:18)
    at Object.handler (Configurator.js:115:14)
    at s.constructor.trigger (Base.js:83:17)
    at BlockType.js:135:16

Since BlockType.js is a part of Neo, I guess this is the right spot to place this issue.

So whatever I do, I am stuck at this point. I cannot remove any condition, the remove buttons do not work anymore.

![Screenshot 2023-10-26 at 12 52 36](https://github.com/spicywebau/craft-neo/assets/90781427/ba9b626c-9ff9-491a-87c0-9c93d26beb26)

The conditions are at `conditionRules` in _neoBlockTypes/*.yml_. I want the field to only show up, when a specific section is met and if the entry on which the field is to be placed has a specific id (`994`).

_neoBlockTypes/*.yml_

```yaml
childBlocks: null
conditions:
  craft\elements\Entry:
    class: craft\elements\conditions\entries\EntryCondition
    conditionRules:
      - class: craft\elements\conditions\entries\SectionConditionRule
        operator: in
        uid: 4b7e459c-db18-43a0-b013-18fbd9a493cd
        values:
          - b7113db4-2571-48c8-a96d-71ce9168ec4d # Pages
      - class: craft\elements\conditions\RelatedToConditionRule
        elementId: "994"
        elementType: craft\elements\Entry
        uid: bb85a607-6d84-4039-98e6-5530fefb9f19
    elementType: craft\elements\Entry
    fieldContext: global
description: ""
enabled: true
field: 58922655-8957-4cbd-b6dc-90e02bba59da # Flexible Components
fieldLayouts:
  d589fef1-2bdd-4bd6-acd1-c430f16e6c52:
    tabs:
      - elementCondition: null
        elements:
          - elementCondition: null
            fieldUid: bf3efd51-94e6-4e89-a5bc-cd2a21cd0604 # Block: Constructions Map
            instructions: null
            label: null
            required: false
            tip: null
            type: craft\fieldlayoutelements\CustomField
            uid: 9348031f-8429-489e-ba51-5bfe1d86a63e
            userCondition: null
            warning: null
            width: 100
        name: Content
        uid: a2d23f6d-0fc7-40ac-98ea-676617118257
        userCondition: null
      - elementCondition: null
        elements:
          - elementCondition: null
            fieldUid: c85a07cb-ab74-4bd2-9c18-475ed7ed215b # Component Settings
            instructions: null
            label: null
            required: false
            tip: null
            type: craft\fieldlayoutelements\CustomField
            uid: 39cfa50b-c1ea-48a6-9aa6-758303ba38e8
            userCondition: null
            warning: null
            width: 100
        name: Settings
        uid: c5462a9c-4683-4ecb-a5c2-6498b9b1efe4
        userCondition: null
group: b0ad1ee1-f612-4db7-a4a7-18b8af459f22 # Columns
groupChildBlockTypes: true
handle: blockConstructionsMap
icon: null
ignorePermissions: true
maxBlocks: 0
maxChildBlocks: 0
maxSiblingBlocks: 0
minBlocks: 0
minChildBlocks: 0
minSiblingBlocks: 0
name: "Block: Constructions Map"
sortOrder: 34
topLevel: true
```

### Steps to reproduce

1. Create a new Neo field
2. Add a new Block Type
3. Choose any `Name` and `handle`
4. Expand „Advanced”
5. Add at least 2 conditions for „Entry Condition” (also same behaviour for „Global Set Condition” so I assume it's the same for all possible conditions). It does not matter which condition.

![Screenshot 2023-10-26 at 13 00 29](https://github.com/spicywebau/craft-neo/assets/90781427/7d01c276-e6d0-406d-b618-50a30ab31231)

To **undo**, remove the block type and run `./craft gc/run`.

### Expected behaviour

I would expect that:

1. The second condition is editable after saving
2. All conditions can be removed when I click the remove button (X) on the right

### Neo version

3.9.7

### Craft CMS version

4.5.9

### What is the affected Neo field's propagation method?

Only save blocks to the site they were created in

### Does this issue involve templating, and if so, is eager-loading used?

This is not a templating issue

**Comment by ttempleton** [comment]
Thanks for reporting this - fixed now in 3.9.8.

Depending on how you're using the 'related to' entry condition, it may be worth noting that the block type won't be dynamically made available or not available when changing the value of an entries field on the same entry as the Neo field; this is the subject of feature request #778.

**Comment by fvwanja** [comment]
Thank you very much @ttempleton ! That may have been the fastest bug resolving that I ever witnessed on the internet 👍
